wp-content/themes/access/controllers/single-location.php

```diff
@@ -29,15 +29,19 @@ public function __construct() {
     enqueue_inline('google-tag-manager');
     enqueue_script('main');
 
-    $alerts = Timber::get_posts(array(
-      'post_type' => 'alert',
-      'posts_per_page' => -1
-    ));
-
-    $this->alerts = array_filter($alerts, function($p) {
-      $flags = ['locations', 'single'];
-      return count(array_intersect(array_values($p->custom['location']), $flags)) === count($flags);
-    });
+    if (get_field('alert')) {
+      $this->alerts = get_field('alert');
+    } else {
+      $alerts = Timber::get_posts(array(
+        'post_type' => 'alert',
+        'posts_per_page' => -1
+      ));
+
+      $this->alerts = array_filter($alerts, function($p) {
+        $flags = ['locations', 'single'];
+        return count(array_intersect(array_values($p->custom['location']), $flags)) === count($flags);
+      });
+    }
 
     return $this;
   }
```

Comment:
I think you need to update your branch... These changes should already be there.